guide/start/policies.md

```diff
@@ -9,169 +9,423 @@ children:
-autoscaler.metricLowerBound               10                                                                   The lower bound of the monitored metric. Below this the policy will resize down   
-autoscaler.metricUpperBound               100                                                                  The upper bound of the monitored metric. Above this the policy will resize up   
-autoscaler.minPeriodBetweenExecs          100ms                                                                   
-autoscaler.minPoolSize                    1                                                                       
-autoscaler.poolColdSensor                 Sensor: resizablepool.cold (java.util.Map)                              
-autoscaler.poolHotSensor                  Sensor: resizablepool.hot (java.util.Map)                               
-autoscaler.poolOkSensor                   Sensor: resizablepool.cold (java.util.Map)                              
-autoscaler.resizeDownIterationIncrement   1                                                                    Batch size for resizing down; the size will be decreased by a multiple of this value   
-autoscaler.resizeDownIterationMax         2147483647                                                           Maximum change to the size on a single iteration when scaling down   
-autoscaler.resizeDownStabilizationDelay   0ms                                                                     
-autoscaler.resizeOperator                 org.apache.brooklyn.policy.autoscaling.AutoScalerPolicy$3@387a7e10      
-autoscaler.resizeUpIterationIncrement     1                                                                    Batch size for resizing up; the size will be increased by a multiple of this value   
-autoscaler.resizeUpIterationMax           2147483647                                                           Maximum change to the size on a single iteration when scaling up   
-autoscaler.resizeUpStabilizationDelay     0ms                                               
+<pre>
+ Id:              tOpMeYYr   
+ Name:            Tomcat Server   
+ Status:          ON_FIRE   
+ ServiceUp:       false   
+ Type:            org.apache.brooklyn.entity.webapp.tomcat.TomcatServer   
+ CatalogItemId:   org.apache.brooklyn.entity.webapp.tomcat.TomcatServer:0.0.0.SNAPSHOT   
+</pre>
+
+<!-- COMMENT:
+You can view its activity, to see the call to restart, using:
+
+{% highlight bash %}
+br application "Tomcat Cluster" entity "Cluster" entity "Tomcat Server" activity
 {% endhighlight %}
 
+TODO Why doesn't the restart() show in the activity view?!
+-->
+
+After the given `stabilizationDelay`, the Tomcat server will be automatically restarted.
+
+
+## Cluster auto-replace
+
+The cluster has a `ServiceReplacer` policy. This attaches to a DynamicCluster and replaces a 
+failed member; if this fails, it sets the Cluster state to on-fire.
```

Comment:
I don't think 'on-fire' has been explained anywhere before - perhaps the section above about failure is a good place to do this?